tests/common/platform/interface_utils.py

```diff
@@ -203,3 +203,17 @@ def get_dpu_npu_ports_from_hwsku(duthost):
             dpu_npu_port_list.append(intf)
     logging.info(f"DPU NPU ports in hwsku.json are {dpu_npu_port_list}")
     return dpu_npu_port_list
+
+
+def check_interface_running_status(duthost, interface):
```

Comment:
@vvolam sonic host as the API `links_status_up` which you can use instead

Reply:
Agree. Prefer to use the library API.

Reply:
@prgeor @dgsudharsan I used is_interface_status_up from latest code library which checks for interface status.